watertap/unit_models/gac.py

```diff
@@ -222,7 +223,7 @@ def eq_isothermal(b, t):
             self.flowsheet().config.time,
             doc="Isothermal assumption for absorbed contaminant",
         )
-        def eq_isothermal_adsorbate(b, t):
+        def eq_isothermal_adsorbed_contam(b, t):
```

Comment:
While we're back here, these constraints should have doc strings.

Reply:
@adam-a-a I'm confused here, do they not already have doc strings? Maybe some clarification would help

Reply:
False alarm. =)